helm-chart/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v2
 name: hedera-json-rpc-relay
 description: Helm chart deployment of the hashgraph/hedera-json-rpc-relay
 type: application
-version: 0.20.0-SNAPSHOT
-appVersion: "0.20.0-SNAPSHOT"
+version: 0.20.0-rc1
+appVersion: 0.20.0-rc1
```

Comment:
I think helm chart recommend quotes for `appVersion` although not required. https://helm.sh/docs/topics/charts/

Reply:
since I used the .bump-version.js script, what it does is basically, loading the yaml, editing it and then rewriting it again...
```
....
const yaml = require('js-yaml');
....
const helmChartPath = 'helm-chart/Chart.yaml';
let doc = yaml.load(fs.readFileSync(helmChartPath, 'utf-8'));
doc.version = newVersion;
doc.appVersion = `${newVersion}`;
fs.writeFileSync(helmChartPath, yaml.dump(doc));
console.log(helmChartPath);
```

So it is using the yaml standard of `js-yaml` 

I guess I could refactor the script to use regex and replace, so it does not rewrites the whole yaml, and we can have more control over how to write the version, but I thought it would be nice to have it standardized.

Reply:
upon further read, I see what you said:
`Wrapping the version in quotes is highly recommended`
...so I will modify the file manually this time and put a PR for a new version of the `.bump-version.js` script.